.github/workflows/binaries.yml

```diff
@@ -107,44 +85,20 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: ${{ env.ARTIFACT_NAME }}
-
       - name: Uncompress Artifact
         run: tar xvf ${{ env.ARTIFACT_NAME }}
-
       - name: Export JSON Schema
         run: |
           ./stellar xdr types schema --type TransactionEnvelope > ${{ env.JSON_SCHEMA_NAME }}
-
       - name: Upload Artifact
-        uses: actions/upload-artifact@v4
+        uses: ./.github/actions/artifact-upload
```

Comment:
we don't really need this anymore, now that stellar/rs-stellar-xdr is publishing the json schema.

Reply:
Should JSON schema be deleted entirely then?

Reply:
yep! Somehow I missed this comment, sorry about that!